changelog.txt

```diff
@@ -1,3 +1,14 @@
+1.1.1-preview
+=============
+This release marks the third preview of the library which brings in several features and changes -
+- Functionality on Unix with Xamarin.Mac Full profile and Mono (#451)

```

Comment:
Not sure what this means. This was simply a bug fix where some API call was throwing an exception on mac when running core clr apps

Reply:
Since it's a bug fix, i just removed it